code/modules/paperwork/handlabeler.dm

```diff
@@ -63,7 +63,8 @@
 		if(!str || !length(str))
 			user << SPAN_NOTICE("Invalid text.")
 			return
-		label = str
+
+		label = replacetext(str, "&#255;", "�")
```

Comment:
It's better to disallow using of cyrillic symbols in label.
Labels are added to the object name some symbols can't work there at all.